tests/Microsoft.Bot.Builder.AI.LUIS.Tests/LuisApplicationTests.cs

```diff
@@ -47,20 +47,46 @@ public void LuisApplication_Serialization()
         [TestMethod]
         public void LuisApplication_Configuration()
         {
+            // Arrange
             var service = new LuisService
             {
                 AppId = Guid.NewGuid().ToString(),
                 SubscriptionKey = Guid.NewGuid().ToString(),
                 Region = "westus"
             };
 
+            // Act
             var model = new LuisApplication(service);
 
+            // Assert
             Assert.AreEqual(service.AppId, model.ApplicationId);
             Assert.AreEqual(service.SubscriptionKey, model.EndpointKey);
             Assert.AreEqual(service.GetEndpoint(), model.Endpoint);
         }
 
+        [TestMethod]
+        public void ListApplicationFromLuisEndpoint()
+        {
+            // Arrange
+            var endpoint = "https://westus.api.cognitive.microsoft.com/luis/v2.0/apps/b31aeaf3-3511-495b-a07f-571fc873214b?verbose=true&timezoneOffset=-360&subscription-key=048ec46dc58e495482b0c447cfdbd291&q=";
```

Comment:
Can you make this a nonsensicle key? Or put a comment in? I don't want to have to re-write github history because a key has been stored in here.